### DISCUSSION TITLE
Two remote repos for same project with different files

### DISCUSSION THREAD

**Discussion by ShaCP** [community/community#23777] [New to GitHub]
<p>How do I set up so that I can have two identical repos except one doesn’t have the source code? Just the production build. I want to update the one with the source code continually and then once I feel like making a new build, update the prod repo. What’s the best workflow for this? In layman’s terms for a beginner.</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi shacp! You don’t want the prod repo to have any source code at all, just the compiled binaries?</p>
<p>You could create a totally separate repo that just contains a README and use that for your prod repo, and attach each build as a Release?</p>
<p><a href="https://help.github.com/en/github/administering-a-repository/managing-releases-in-a-repository#creating-a-release" class="onebox" target="_blank">https://help.github.com/en/github/administering-a-repository/managing-releases-in-a-repository#creating-a-release</a></p>
<p>Doing this seems perhaps a little counterproductive - traditionally Releases are a snapshot of the code in the repository at a specific point.</p>
<p>You wouldn’t perhaps prefer to have a dev repo where you work on your code as normal, and then a prod repo where you push up only the <code>master</code> branch when you are ready to make a new production build? You could do that by creating multiple remotes from your local copy:</p>
<p><a href="https://help.github.com/en/github/using-git/about-remote-repositories#creating-remotes" class="onebox" target="_blank">https://help.github.com/en/github/using-git/about-remote-repositories#creating-remotes</a></p>
<p>So then you could use:</p>
<p><code>git remote add origin  &lt;DEV_REPO_URL&gt; </code></p>
<p>…and push to origin when working on your code most of the time. And then add:</p>
<p><code>git remote add prod  &lt;PROD_REPO_URL&gt; </code></p>
<p>…and only occasionally push to prod. (And then make a release as well, if you like.)</p>
<p>I’d like to hear what other people think of adding Releases to a repo that doesn’t contain the code. Is it an absolute no no, or an acceptable practice?</p>
